### DISCUSSION TITLE
Version compatibility

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5386] [Q&A]
my  GKE got auto upgraded to version 1.20  and Strimzi Kafka is failing to start. is there any known compatibility issue with this version?  any fix you may suggest?

**Comment by scholzj** [comment]
I'm not aware of anything. But without knowing the Strimzi version you use or seeing any logs it is hard to say. Strimzi versions go years back, so there are definitely some which will not work. But the latest versions should not have any issues.

**Reply by sreejesh123** [reply]
my operator - 0.20.1
kafka -kafka-2.6.0

Error I will get it soon. now going for a reinstall to see if that will solve the issue. possible issue may be with a GKE upgrade with no heads up.
